### PR TITLE
fix: make devpool PR msg CC compliant

### PR DESCRIPTION
**Pull request by rndquu**
We've got a feature of opening a PR with partner URL to the [devpool repo
](https://github.com/ubiquity/devpool-directory) when a new partner installs the bot

The [commit message](https://github.com/ubiquity/devpool-directory/pull/667) is not conventional commits compliant 

This PR fixes it

### CODE REVIEW

File: src/handlers/pull-request/create-devpool-pr.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const createDevPoolPR = async () => {
     owner: devPoolOwner,
     repo: devPoolRepo,
     path,
-    message: `Add ${repository.full_name} to repo`,
+    message: `feat: add ${repository.full_name} to repo`,
```
Comment (pavlovcik) [inline src/handlers/pull-request/create-devpool-pr.ts:68]:
```suggestion
    message: `feat: add ${repository.full_name}`,
```

**Review by pavlovcik — APPROVED** [review]

**Comment by pavlovcik** [comment]
I wasn't actually actively aware of the feature in the first place. Its cool but I also feat that pull requests will pile up and be neglected. Perhaps its better if the partner manually opens the pull request instead of it being automatic? Perhaps not 100% of the partners want to be added, but with this approach, 100% will have to be reviewed in the piled up pull requests.

**Comment by rndquu** [comment]
> I wasn't actually actively aware of the feature in the first place. Its cool but I also fear that pull requests will pile up and be neglected. Perhaps its better if the partner manually opens the pull request instead of it being automatic?
> 
> Perhaps not 100% of the partners want to be added, but with this approach, 100% will have to be reviewed in the piled up pull requests.

We discussed it here https://github.com/ubiquity/ubiquibot/issues/278

We don't have piled up PRs right now so I would leave it "as is" for now and fix it later (if it's really necessary)

**Comment by pavlovcik** [comment]
> > I wasn't actually actively aware of the feature in the first place. Its cool but I also fear that pull requests will pile up and be neglected. Perhaps its better if the partner manually opens the pull request instead of it being automatic?
> 
> > 
> 
> > Perhaps not 100% of the partners want to be added, but with this approach, 100% will have to be reviewed in the piled up pull requests.
> 
> 
> 
> We discussed it here https://github.com/ubiquity/ubiquibot/issues/278

Should we close that issue if it's functional?

> We don't have piled up PRs right now so I would leave it "as is" for now and fix it later (if it's really necessary)

Ok

**Comment by rndquu** [comment]
> Should we close that issue if it's functional?

@Draeieg I think we need to QA this [issue](https://github.com/ubiquity/ubiquibot/issues/278) one more time because it seems to be [working](https://github.com/ubiquity/devpool-directory/pull/667)

**Pull request merged**